VmBackup-BETA-3.23.py

```diff
@@ -480,21 +484,21 @@ def main(session):
         if config_specified:
             status_log_end(server_name, 'ERROR,%s' % summary)
             # MAIL_ENABLE: optional email may be enabled by uncommenting out the next two lines
-            #send_email(MAIL_TO_ADDR, 'ERROR VmBackup.py', status_log)
+            #send_email(MAIL_TO_ADDR, 'ERROR ' + os.uname()[1] + ' VmBackup.py', status_log)
```

Comment:
Would consider assigning `os.uname()[1]` its own variable function that makes it more obvious what it is actually returning.